main/libgit2/APKBUILD

```diff
@@ -21,6 +21,8 @@ options="!check" # FIXME some tests fails
 builddir="$srcdir/$pkgname-$pkgver"
 
 # secfixes:
+#   0.28.0:
```

Comment:
it should point to `0.28.1-r0` where this fix appears

Reply:
fixed